rust/worker/src/execution/orchestration/metadata.rs

```diff
@@ -265,11 +263,44 @@ impl MetadataQueryOrchestrator {
             logs,
             self.query_ids.clone(),
             self.where_clause.clone(),
-            self.offset,
+        );
+
+        let op = FilterOperator::new();
+        let task = wrap(op, input, ctx.receiver());
+        match self.dispatcher.send(task, Some(Span::current())).await {
+            Ok(_) => (),
+            Err(e) => {
+                // Log an error - this implies the dispatcher was dropped somehow
```

Comment:
we should fail the request at this point

Reply:
This is addressed in the new `GetOrchestrator` in a later PR.